src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1083,7 +1087,7 @@ static void write_action(void* gt, grpc_error* error) {
 static void write_action_end_locked(void* tp, grpc_error* error) {
```

Comment:
This function is the callback from `grpc_endpoint_write()` and therefore can't go away, but the `_locked()` suffix is probably no longer appropriate, since it acquires the lock itself instead of being called when the call is already held.

In general, any closure callback function ending in `_locked()` should either be removed (by moving its code into the place where the closure was called from) or have its `_locked()` suffix removed.

Reply:
Done!